k4FWCore/components/IOSvc.h

```diff
@@ -58,11 +58,17 @@ class IOSvc : public extends<Service, IIOSvc, IIncidentListener> {
 protected:
   Gaudi::Property<std::vector<std::string>> m_collectionNames{
       this, "CollectionNames", {}, "List of collections to read"};
-  Gaudi::Property<std::vector<std::string>> m_readingFileNames{this, "input", {}, "List of files to read"};
-  Gaudi::Property<std::string>              m_writingFileName{this, "output", {}, "List of files to write output to"};
+  Gaudi::Property<std::vector<std::string>> m_readingFileNamesDeprecated{this, "input", {}, "List of files to read"};
+  Gaudi::Property<std::vector<std::string>> m_readingFileNames{this, "Input", {}, "List of files to read"};
+  Gaudi::Property<std::string> m_writingFileNameDeprecated{this, "output", {}, "List of files to write output to"};
+  Gaudi::Property<std::string> m_writingFileName{this, "Output", {}, "List of files to write output to"};
```

Comment:
Yes, I mention this in the PR. I think it's easy to find most places where people use functional algorithms since there are not so many and change it so that this shouldn't appear for a very long time.

Reply:
Right, I forgot it already was in the description